### ISSUE TITLE
padding(0,1) in pytorch,how to realize this function in tensorflow

### ISSUE THREAD

**Issue by lxyzler** [Oldpan/faceswap-pytorch#6]
such as "torch.nn.MaxPool2d(kernel_size=2, stride=(2,1), padding=(0,1))"
